saleor/graphql/core/utils/error_codes.py

```diff
@@ -96,10 +96,12 @@ def get_error_code_from_error(error) -> str:
         return "required"
     if code in ["unique", "unique_for_date"]:
         return "unique"
-    if code in DJANGO_VALIDATORS_ERROR_CODES or code in DJANGO_FORM_FIELDS_ERROR_CODES:
+    if (
+        code is None
+        or code in DJANGO_VALIDATORS_ERROR_CODES
+        or code in DJANGO_FORM_FIELDS_ERROR_CODES
+    ):
         return "invalid"
     if isinstance(code, Enum):
         code = code.value
-    if code not in saleor_error_codes:
-        return "invalid"
```

Comment:
I think the changes look good but I'm wondering if we need to remove this part. Can you provide an example of how you raise those custom errors in your plugin? I think if you used an enum to represent errors, the code would be handled in the previous condition and removing this one wouldn't be necessary? I mean for example this usage:

```
code=AccountErrorCode.DELETE_OWN_ACCOUNT
```

Reply:
@maarcingebala A plugin exposes a playground at /plugins/plugin-id/ and has a `ModelMutation` with an `error_type_class` of a custom error created with a custom enum created, always says "expected a value of type `error_type_class_here`, received invalid"

Reply:
This part restricts the codes to be from the internal saleor error codes, we want to add custom ones in this case

Reply:
The workaround currently is to use any random error code from saleor itself, but this does not help when extending the dashboard because you usually show a message depending on the code

Reply:
So yes, your example works perfectly, because AccountErrorCode is in `saleor_error_codes` but `MyPluginErrorCode` is not

Reply:
I've checked and you're right. We can remove this part then, but in this case, we should also remove `saleor_error_codes` and `SALEOR_ERROR_CODE_ENUMS` which won't be needed anymore.

Reply:
Great then, I haven't seen if they were used elsewhere, I'll check and if so, I'll remove them in a few minutes

Reply:
I've currently pushed the commit, if it passes, you're good to go